types/convict/convict-tests.ts

```diff
@@ -1,6 +1,6 @@
 import * as convict from 'convict';
-import * as validator from 'validator';
```

Comment:
these imports are irrelevant so I removed them to remove the dependency between the convict package and the validator/js-yaml type definitions

Reply:
Are you sure that this usage won't break this test https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0a4230ce80fdd64586301e88bce0666660f2b37e/types/convict/convict-tests.ts#L69 ?

Reply:
yes since these tests aren't actually being run but only type-checked

Reply:
If it is not ran, I think this is better to totally remove the usage instead of making it `void 0`, that would be less confusing for the next contributor